config/dotfiles/zshenv

```diff
@@ -19,7 +19,7 @@ get_script_dir() {
 }
 
 readonly PARENT_DIR="$(get_script_dir)"
-readonly REPO_ROOT="$PARENT_DIR/../.."
+readonly BATTLESTATION_ROOT="$PARENT_DIR/../.."
```

Comment:
When we moved internal scripts to use zsh `REPO_ROOT` began to conflict with another environment variable.